framework/src/base/Moose.C

```diff
@@ -893,6 +905,7 @@ addActionTypes(Syntax & syntax)
 "(add_postprocessor)"
 "(add_vector_postprocessor)"
 "(add_aux_kernel, add_bc, add_damper, add_dirac_kernel, add_kernel, add_dg_kernel, add_scalar_kernel, add_aux_scalar_kernel, add_indicator, add_marker)"
+"(add_control)"
```

Comment:
Just curious, does this task need to be at this location or can it be moved around if needed? The other user_objects are added way earlier, what makes them available for use (test for existence) in other objects. It does feel like CL items "should" be added near the end but I don't think they actually have a dependency that forces them here.